lib/sippy_cup/g711.rb

```diff
@@ -0,0 +1,36 @@
+# ffi interface to freeswitch's g711
+require 'ffi'
+
+module SippyCup
+  module G711
+    extend FFI::Library
+    ffi_lib 'spandsp'
```

Comment:
This new dependency on spandsp needs a couple of things:

1. Document how and where a person can get spandsp, if they don't already have it
2. Ensure sippy_cup continues to work the way it did before if spandsp is not available